packages/client/src/i18n/messages/views/userSetup.ts

```diff
@@ -302,7 +308,7 @@ const messagesToDefine: IUserSetupMessages = {
     id: 'user.profile.auditList.unAssigned'
   },
   correctedAuditAction: {
-    defaultMessage: 'Corrected',
+    defaultMessage: 'Corrected Record',
```

Comment:
This must be updated in Farajaland. The copy in core is only a fallback

Reply:
Yes this has been updated on farajaland 
see PR : - https://github.com/opencrvs/opencrvs-farajaland/pull/418